build.sbt

```diff
@@ -289,6 +289,15 @@ lazy val doobie  = project
   )
   .dependsOn(core, derivation, env % Test, zioInterop % Test)
 
+lazy val examples = project
+  .in(file("examples"))
+  .settings(
+    libraryDependencies ++= List(doobieCore, derevo, monix),
+    defaultSettings,
+    name := "tofu-examples",
```

Comment:
examples should be skipped during publish, use noPublishSettinga

Reply:
```suggestion
    name := "tofu-examples",
    noPublishSettings,
```